File: python/phase 1-condtional_thinking/q9_vowel_consonant.py
"""
Phase 1: Conditional Thinking (If-Else, Boolean Logic)
Level 1: Simple Conditions (Getting started)
Question 9: Take a character and check if it's a vowel or consonant.

Mentor Tips (Python 3.14+):
- Check if character (converted to lowercase) is one of 'a', 'e', 'i', 'o', 'u'.
"""

def solve(char: str) -> str:
    # TODO: Implement this method to solve the question
    if char.lower() in ['a', 'e', 'i', 'o', 'u']:
        return "vowel"
    else:  
        return "consonant"

def main():
    char = input("Enter a single letter: ")
    result = solve(char)
    print(f"Result: {result}")

if __name__ == "__main__":
    main()

File: java/src/phase 1-conditional_thinking_java/Q9VowelConsonant.java
import java.util.Scanner;

/**
 * Phase 1: Conditional Thinking (If-Else, Boolean Logic)
 * Level 1: Simple Conditions (Getting started)
 * Question 9: Take a character and check if it's a vowel or consonant.
 * 
 * Mentor Tips (Java 26):
 * - Check if character (converted to lowercase) is one of 'a', 'e', 'i', 'o', 'u'.
 */
public class Q9VowelConsonant {

    public static String solve(char charVal) {
        // TODO: Implement this method to solve the question
        return "";
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a single letter: ");\n        char val = scanner.next().charAt(0);\n        System.out.println("Result: " + solve(val));
        scanner.close();
    }
}
